docs/metrics.asciidoc

```diff
@@ -14,6 +14,12 @@ You can adjust the interval with the setting <<config-metrics-interval>>.
 
 The metrics will be stored in the `apm-*` index and have the `processor.event` property set to `metric`.
 
+NOTE: Dedicated JVM metrics views are available since Elastic stack version 7.1.
+Since 7.5, metrics are aggregated separately for each JVM, relying on the ID of the underlying system- either container ID (where applicable) or hostname.
```

Comment:
This should be an em dash instead of en dash:
```suggestion
Since 7.5, metrics are aggregated separately for each JVM, relying on the ID of the underlying system -- either container ID (where applicable) or hostname.
```

Reply:
And a third recommendation in case you like both suggestions:

```suggestion
Starting in 7.5, metrics are aggregated separately for each JVM, relying on the ID of the underlying system -- either container ID (where applicable) or hostname.
```